test/e2e/framework/log_test.go

```diff
@@ -144,141 +132,56 @@ k8s.io/kubernetes/test/e2e/framework_test.glob..func1.5()
 	log_test.go:67
 k8s.io/kubernetes/test/e2e/framework_test.runTests()
 	log_test.go:47` + commonOutput,
-			failure: `of course it's not equal...
+			Failure: `of course it's not equal...
 Expected
     <int>: 0
 to equal
     <int>: 1`,
-			stack: `k8s.io/kubernetes/test/e2e/framework_test.glob..func1.5()
+			Stack: `k8s.io/kubernetes/test/e2e/framework_test.glob..func1.5()
 	log_test.go:67
 k8s.io/kubernetes/test/e2e/framework_test.runTests()
 	log_test.go:47
 `,
 		},
-		testResult{
-			name: "[Top Level] log error",
-			output: `INFO: before
-FAIL: hard-coded error
-Unexpected error:
+		output.TestResult{
+			Name: "[Top Level] log error",
+			Output: `INFO: before
+INFO: Unexpected error: hard-coded error: 
     <*errors.errorString>: {
         s: "an error with a long, useless description",
     }
-    an error with a long, useless description
-occurred
+FAIL: hard-coded error: an error with a long, useless description
```

Comment:
This is an improvement. But after discussing it further on #sig-testing, I think I'll try to make it even better:

```
FAIL: <short failure message>
<details>
```

In this case:
```
FAIL: Unexpected error: hard-coded error: an error with a long, useless description
    <*errors.errorString>: {
        s: "an error with a long, useless description",
    }
```